README.md

```diff
@@ -38,9 +38,9 @@ In the meantime, it's best to install the pre-release `0.1.0-10`.
 
 ## Get started
 
-Built with [ng-packagr] and conforming to the [Angular Package Format].
```

Comment:
still built using ng-packagr

Reply:
Really? Then I made a huge tiny mistake. I thought the idea here was to use the official `angular-cli` itself to build the library format instead of the community projects like `ng-packagr` or `angular-librarian`. And I thought we moved away from `angular-librarian` to `ng-packagr` a while back (maybe when Angular 6 came out?)

So what am I missing?

Reply:
Yeah I made the PR to move this to ng-packagr so you were for sure using it.

ng-packagr is still used to build the library and its as close to official as we're going to get right now and is mentioned in the [angular-cli libraries doc](https://github.com/angular/angular-cli/wiki/stories-create-library) and has commits from the angular core team.

however the rest of the project (demo application) was using the setup from angular-librarian still and had old versions of typescript, webpack, and weird tsconfig settings that were getting in the way of the build. This moved the demo application to angular-cli getting all of that out of the way

Reply:
Yeah, Angular CLI uses ng-packagr under the hood at the moment, so technically it is still built with ng-packagr. I think it is better to say that it is build using Angular CLI though (as you did), as using ng-packagr is  an implementation detail of Angular CLI.